src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -111,48 +122,119 @@ static CallingConvention getCallingConvention(FunctionPointerCallingConventionSy
+                                }
+                                return CallingConvention.Default;
 
                             case { CallingConventions: var specifiers }:
-                                // PROTOTYPE(func-ptr): Handle multiple specifiers
-                                foreach (var convention in specifiers)
+                                // More than one identifier case
+                                checkUnmanagedSupport(compilation, callingConventionSyntax.ManagedOrUnmanagedKeyword.GetLocation(), diagnostics);
+                                foreach (var specifier in specifiers)
                                 {
-                                    _ = reportBadConventionAndReturn(convention.Name.ValueText, convention.Name, diagnostics);
+                                    CustomModifier? modifier = handleIndividualUnrecognizedSpecifier(specifier, compilation, diagnostics);
+                                    if (modifier is object)
+                                    {
+                                        customModifiers.Add(modifier);
+                                    }
                                 }
 
-                                return CallingConvention.Default;
+                                return CallingConvention.Unmanaged;
                         }
 
                     case var unexpected:
                         throw ExceptionUtilities.UnexpectedValue(unexpected);
                 }
 
-                static CallingConvention reportBadConventionAndReturn(string givenConvention, SyntaxNodeOrToken node, DiagnosticBag diagnostics)
+                static CallingConvention handleSingleConvention(FunctionPointerUnmanagedCallingConventionSyntax specifier, CSharpCompilation compilation, ArrayBuilder<CustomModifier> customModifiers, DiagnosticBag diagnostics)
+                {
+                    checkUnmanagedSupport(compilation, specifier.GetLocation(), diagnostics);
+                    var modifier = handleIndividualUnrecognizedSpecifier(specifier, compilation, diagnostics);
+                    if (modifier is object)
+                    {
+                        customModifiers.Add(modifier);
+                    }
+                    return CallingConvention.Unmanaged;
+                }
+
+                static CustomModifier? handleIndividualUnrecognizedSpecifier(FunctionPointerUnmanagedCallingConventionSyntax specifier, CSharpCompilation compilation, DiagnosticBag diagnostics)
+                {
+                    string specifierText = specifier.Name.ValueText;
+                    if (string.IsNullOrEmpty(specifierText))
+                    {
+                        return null;
+                    }
+
+                    string typeName = "CallConv" + specifierText;
+                    var metadataName = MetadataTypeName.FromNamespaceAndTypeName("System.Runtime.CompilerServices", typeName);
```

Comment:
>FromNamespaceAndTypeName [](start = 56, length = 24)

I think we want to use ```useCLSCompliantNameArityEncoding: true``` for this name. We don't want to pick up a generic type that is missing the arity suffix in the name. It would be good to test for this.